apps/settings/js/panels/screen_lock/screen_lock.js

```diff
@@ -1,38 +1,25 @@
-/* global SettingsListener */
 define(function(require) {
   'use strict';
 
-  var SettingsService = require('modules/settings_service');
+  var DialogService = require('modules/dialog_service');
+  var SettingsListener = require('shared/settings_listener');
 
   var ScreenLock = function ctor_screenlock() {
     return {
-      _panel: null,
-
       _settings: {
         passcodeEnabled: false,
         lockscreenEnabled: false
       },
 
-      _getAllElements: function sl_getAllElements() {
-        this.screenlockPanel = this._panel;
-        this.lockscreenEnable = this._panel.querySelector('.lockscreen-enable');
-        this.passcodeEnable = this._panel.querySelector('.passcode-enable');
-        this.passcodeEditButton = this._panel.querySelector('.passcode-edit');
-      },
+      onInit: function sl_onInit(elements) {
```

Comment:
as above response, it's a convention to make html elements isolated and easier to be identified.